query/security/wcpm/src/main/java/org/codice/ddf/admin/security/wcpm/actions/discover/GetAuthTypes.java

```diff
@@ -37,11 +37,12 @@ public GetAuthTypes(ConfiguratorFactory configuratorFactory) {
 
     @Override
     public ListField<AuthType> performAction() {
-        // TODO: tbatie - 4/14/17 - Check the backend to see if these services are running
-        return new ListFieldImpl<>(AuthType.class).addAll(Arrays.asList(AuthType.BASIC_AUTH,
+        ListField<AuthType> authTypes = new ListFieldImpl<>(AuthType.class).addAll(Arrays.asList(AuthType.BASIC_AUTH,
```

Comment:
Consider additional implications such as if the auth type isn't running, are we responsible for starting them up? If so, we need an alternative way to add additional descriptions for auth types.